### ISSUE TITLE
[BUG] Saving error in Paper Mario

### ISSUE THREAD

**Issue by AyrisX86** [masterfeizz/DaedalusX64-3DS#14]
Describe the bug
When saving my game in Paper Mario, my game is saved all 4 files as opposed to the 1 I was using

To Reproduce
Save your game then reload the rom (or get a game over) to return to the title screen

Expected behavior
Only the file I was using would be saved

Screenshots
Here are my save files:
![2020-05-05_12-50-37 339_top](https://user-images.githubusercontent.com/64820884/81031383-6dd0db00-8ecf-11ea-8fae-f5c24a2f833c.png)

Device:
 - FW:[11.13.0-45E]
 - Daedalus Version [e.g. 0.2]

Additional context

**Comment by MasterGamingYT** [comment]
How do you even save?

**Comment by rustycruxian** [comment]
You click the lower screen and press save state and choose which slot you want to save in

**Comment by AyrisX86** [comment]
I’m referring to the ingame save functionality. Jump into a Save Block and
you’ll get a prompt to save. My main concern is games like Donkey Kong 64
since it has memory leakage issues and if played for too long without
reloading will simply crash.

**Comment by AyrisX86** [comment]
> How do you even save? Doesn't it reset after a reboot?

No, after you first get control of Mario there are blocks you can hit with the letter S on them. They let you save. The save system does actually work, but the file is saved to all four of the game's save native save files, not the single one I was playing. The emulator's savestate function isn't affected by this.
